.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+jdk:
+- oraclejdk8
+before_install:
+- export | grep SAUCE_
+script: "./gradlew -q -Pversion=$TRAVIS_JOB_NUMBER build publishPlugins"
+env:
+  global:
+  - SAUCE_USERNAME=sauce_examples
+  matrix:
+  - secure: DfmbwK6MErbJTEs2AGdn1UNQwnC4Hdz0S0VPkiwkjKkZSL1zjBPTA+Zm24DM1sgGxIH0P1YYwjZnr7+EX5BWOqxMT/E8FsuMw79qVucKnI3Oh2axBAYttcaIYcw0ue4DXuxt3JifTo3ivWDnLhXet4Cn25JHS5jVLqTXjwwgDFnND5yEFJc34UWowSIAcUs9Shv96KokRVFVpkmFSE6Gin+8o4P7CpYAOaNabIZLiVhAMzB2Zk3nAGWnkfbBAB36wx961HzLypP4qrQtDhrAy5wRvLc0k2hOUPscJcsyi2tnGKRL/QGHhdu6wRfKOkuQrOljWzsc3v29OfUZdYposfLNpi/BHGNDg9gG0kTyrHOpdI2GEmY71T3//MevucJH4ZGUJA8cAQa0H+PFMxHpBW7V3ptLnrQOiFa77fgmFvaGjvIShATPyUE6h7XqN0v1CI2m0IngNVJLGi/YawzageMW+TArbTG3LfOwpMGldu3ufB0Bk+mh2gw61zPHx2hYdla1vdxR7pSxbSe+oCpjIZtI3KYKJa6i3kky19kXvyGkqRkKgbYYGYOWXNAacL3JA/OzQaB6Ym0QLUbGCpekWs/829Tnh34a5FgqvWEPhXB9Qta/ZNtqYt9DAdGX09CNSM4jRL7uB1pFU7zqjZkyYKQmLCej8gZrkUqPDJT7Vt0=
```

Comment:
Actually, because this is a generic sequence per YAML spec it doesn't have to be indented [src: [Generic Sequence](http://www.yaml.org/spec/1.2/spec.html#id2802662) ].